src/frontend/package.json

```diff
@@ -28,6 +28,7 @@
     "css-loader": "^0.28.4",
     "csvtojson": "^1.1.7",
     "d3": "^4.9.1",
+    "dimensions": "^0.1.0",
```

Comment:
looks like the only place this is used is commented out - is this intended for future incorporation?